examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,30 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for a binary, and shows
+what type of telemetry data it can produce for certain RPC's with additional CSM
+Labels. The Client accepts configuration from an xDS Control plane as the
+default address that it connects to is "xds:///helloworld:50051", but this can
+be overridden with the command line flag --addr.
+
+## Try it (locally if overwritten xDS Address)
+
+```
+go run server/main.go
+```
+
+```
+go run client/main.go
+```
+
+```
+curl localhost:9464/metrics
```

Comment:
Just a brief mention here that this is the port where the prometheus exporter is outputting metrics data on would be helpful.

Reply:
Done.

Reply:
Please move the comment to be outside the backticks.

Reply:
Ah, done.